api/src/main/java/io/grpc/InternalConfigSelector.java

```diff
@@ -42,7 +42,6 @@
     private final Status status;
     @Nullable
```

Comment:
Also remove this annotation and the one for `getConfig()` for consistency with callOptions.

Reply:
Oops, forgot this was reverted...  Fixe now.